maven/tests/strings_test.bzl

```diff
@@ -0,0 +1,16 @@
+load(":testing.bzl", "asserts", "test_suite")
+load("//maven:utils.bzl", "strings")
+
+def trim_test_default(env):
+    asserts.equals(env, "foo", strings.trim("   \n foo \n"))
+
+def trim_test_custom(env):
+    asserts.equals(env, "foo", strings.trim(" aaa fooa \n", "\n a"))
+
+def contains_test(env):
+    asserts.true(env, strings.contains("foobarbash", "bar"))
+    asserts.false(env, strings.contains("foobarbash", "Bar"))
+
+# Roll-up function.
+def strings_test_suite():
+    test_suite("strings", tests = [trim_test_custom, trim_test_default, contains_test])
```

Comment:
Will you know when you forget to add a test to these lists?

Reply:
Nope.  I'm trying to think of a decent way to test for that.  Haven't yet. Possibly just parsing the files, and enforcing that any test ends in _test (and renaming a few of these), and that it's in these lists.  But there's some turtles all the way down situations here, and there's no reflective power to list all the functions in the file that I know of.